Comanche055/IMU_CALIBRATION_AND_ALIGNMENT.agc

```diff
@@ -419,79 +420,78 @@ FINETIME	INHINT			# RETURNS WITH INTERRUPT INHIBITED
 		TC Q
 
 # Page 432
-# PROGRAM NAME:  OPTIMUM PRELAUNCH ALIGNMENT CALIBRATION
-# DATE:  NOVEMBER 2 1966
-# BY:  GEORGE SCHMIDT IL 7-146 EXT. 126
+# PROGRAM NAME-OPTIMUM PRELAUNCH ALIGNMENT CALIBRATION
+# DATE- NOVEMBER 2 1966
+# BY- GEORGE SCHMIDT IL 7-146 EXT. 126
 # MOD NO 3
-#
 # FUNCTIONAL DESCRIPTION
-#
-# THIS SECTION CONSISTS OF PRELAUNCH ALIGNMENT AND GYRO DRIFT TESTS
+
+# THIS	SECTION CONSISTS OF PRELAUNCH ALIGNMENT AND GYRO DRIFT TESTS
 # INTEGRATED TOGETHER TO SAVE WORDS.  COMPASS IS COMPLETELY RESTART
 # PROOFED EXCEPT FOR THE FIRST 30 SECONDS OR SO.  PERFORMANCE TESTS OF
 # THE IRIGS IS RESTART PROOFED ENOUGH TO GIVE 75 PERCENT CONFIDENCE THAT
 # IF A RESTART OCCURS THE DATA WILL STILL BE GOOD.  GOOD PRACTICE TO RECYCL
-# WHEN A RESTART OCCURS UNLESS IT HAPPENS NEAR THE END OF A TEST -- THEN WAIT
+# WHEN A RESTART OCCURS UNLESS IT HAPPENS NEAR THE END OF A TEST-THEN WAIT
 # FOR THE DATA TO FLASH.
-#
-# A RESTART IN GYROCOMPASS DURING GYRO TORQUING CAUSES PULSES TO BE LOST.
+# A RESTART IN GYROCOMPASS DURING GYRO TORQUING CAUSES PULSES TO BE LOST
 # THE PRELAUNCH ALIGNMENT TECHNIQUE IS BASICALLY THE SAME AS IN BLOCK 1
 # EXCEPT THAT IT HAS BEEN SIMPLIFIED IN THE SENSE THAT SMALL ANGLE APPROX.
 # HAVE BEEN USED.  THE DRIFT TESTS USE A UNIQUE IMPLEMENTATION OF THE
-# OPTIMUM STATISTICAL FILTER.  FOR A DESCRIPTION SEE E-1973.  BOTH OF THESE
+# OPTIMUM STATISTICAL FILTER.  FOR A DESCRIPTION SEE E-1973.BOTH OF THESE
 # ROUTINES USE STANDARD SYSTEM TEST LEADIN PROCEDURES.  THE INITIALIZATION
-# PROCEDURE THE DRIFT TESTS IS IN THE JDC'S.  THE INITIALIZATION METHOD
+# PROCEDURE THE DRIFT TESTS IS IN THE JDC S.  THE INITIALIZATION METHOD
 # FOR GYROCOMPASS IS AN ERAS LOAD THEN A MISSION PHASE CALL.
-# THE COMPASS ALIGNS TO Z DOWN, X DOWNRANGE, HAS THE CAPABILITY
-# CHANGE AZIMUTH WHILE RUNNING, IS COMPENSATED FOR
-# COMPONENT ERRORS, IS CAPABLE OF OPTICAL VERIFICATION (CSM ONLY).
-#
+# THE COMPASS ALIGNS TO Z DOWN,X DOWNRANGE, HAS THE CAPABILITY
+# CHANGE AZIMUTH WHILE RUNNING , IS COMPENSATED FOR
```

Comment:
Incorrect spacing on multiple lines. Again, why are you adding/removing random characters?